components/eam/src/physics/crm/physpkg.F90

```diff
@@ -1308,64 +1356,26 @@ subroutine tphysbc(ztodt, fsns, fsnt, flns, flnt, &
-  real(r8) :: flx_cnd(pcols)
   real(r8) :: flx_heat(pcols)
-  type(check_tracers_data):: tracerint       ! energy integrals and cummulative boundary fluxes
-  real(r8) :: zero_tracers(pcols,pcnst)
 
   logical   :: lq(pcnst)
 
   character(len=fieldname_len)   :: varname, vsuffix
 
-  !BSINGH - these were moved from zm_conv_intr because they are  used by aero_model_wetdep 
-  real(r8):: mu(pcols,pver) 
-  real(r8):: eu(pcols,pver)
-  real(r8):: du(pcols,pver)
-  real(r8):: md(pcols,pver)
-  real(r8):: ed(pcols,pver)
-  real(r8):: dp(pcols,pver)
-  real(r8):: dsubcld(pcols) ! wg layer thickness in mbs (between upper/lower interface).
-  integer :: jt(pcols)      ! wg layer thickness in mbs between lcl and maxi.    
-  integer :: maxg(pcols)    ! wg top  level index of deep cumulus convection.
-  integer :: ideep(pcols)   ! wg gathered values of maxi.
-  integer :: lengath        ! w holds position of gathered points vs longitude index
-
   real(r8) :: ftem(pcols,pver)         ! tmp space
   real(r8), pointer, dimension(:) :: static_ener_ac_2d ! Vertically integrated static energy
   real(r8), pointer, dimension(:) :: water_vap_ac_2d   ! Vertically integrated water vapor
   real(r8) :: CIDiff(pcols)            ! Difference in vertically integrated static energy
 
-  logical :: l_bc_energy_fix, l_dry_adj, l_tracer_aero, l_st_mac, l_st_mic, l_rad
+  logical :: l_bc_energy_fix, l_dry_adj
   
   logical           :: use_ECPP
   character(len=16) :: MMF_microphysics_scheme
-  real(r8), dimension(pcols,pver) :: mmf_clear_rh ! CRM clear air relative humidity used for aerosol water uptake
-  real(r8), dimension(pcols) :: mmf_qchk_prec_dp  ! CRM precipitation diagostic (liq+ice)  used for check_energy_chng
-  real(r8), dimension(pcols) :: mmf_qchk_snow_dp  ! CRM precipitation diagostic (ice only) used for check_energy_chng
-  real(r8), dimension(pcols) :: mmf_rad_flux      ! CRM radiative flux diagnostic used for check_energy_chng
-  type(crm_ecpp_output_type) :: crm_ecpp_output   ! CRM output data for ECPP calculations
-#if defined( ECPP )
-  ! ECPP variables
-  real(r8),pointer,dimension(:)   :: pblh              ! PBL height (for ECPP)
-  real(r8),pointer,dimension(:,:) :: acldy_cen_tbeg    ! cloud fraction
-  real(r8)                        :: dtstep_pp         ! ECPP time step (seconds)
-  integer                         :: necpp             ! number of GCM time steps in which ECPP is called once
-#endif /* ECPP */
 
   call phys_getopts( l_bc_energy_fix_out    = l_bc_energy_fix    &
                     ,l_dry_adj_out          = l_dry_adj          &
-                    ,l_tracer_aero_out      = l_tracer_aero      &
-                    ,l_st_mac_out           = l_st_mac           &
-                    ,l_st_mic_out           = l_st_mic           &
-                    ,l_rad_out              = l_rad              &
                     ,use_ECPP_out           = use_ECPP           &
```

Comment:
Likewise, can we lose the ECPP and MMF vars here?

Reply:
yup.